services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None):
+
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(user_id: int, username: str = None,
+                password: str = None,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+
+    user = get_user(user_id=user_id)
+
+    if username:
+        user.username = username
+
+    if password:
+        user.set_password(password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
```

Comment:
Don't Repeat Yourself issue: You should call the `save()` method only once if it's possible. Here you can set all the attributes first and then call `save()` method."Good example:\nuser = get_user_model().objects.create_user(\n    username=username, password=password\n)\n\nif first_condition:\n    # some actions\nif second_condition:\n    # some actions\n    \nuser.save()\n\nBad example:\nuser = get_user_model().objects.create_user(\n    username=username, password=password\n)\n\nif first_condition:\n    # some actions\n    user.save()\nif second_condition:\n    # some actions\n    user.save()"